torch/csrc/jit/tracer.cpp

```diff
@@ -100,6 +142,8 @@ Node* recordTrace(std::string op, // TODO: make this a Symbol
   auto state_lock = state->lock();
 
   Node *n = graph->create(stringToSymbol(op));
+  n->setDebugName(getPythonInterpreterStackTrace());
```

Comment:
I don't think this is needed in here - this is the codepath for C++ ops, so it will never return anything interesting (the stack trace will always be empty). Editing `python_function.cpp` should be sufficient

Reply:
We also care about where ATen ops get instantiated though, so we do want to record these paths.

Reply:
Ah right, nevermind. The threads dispatching those function will actually by Python threads, my bad.